server/main.lua

```diff
@@ -343,3 +64,79 @@ ESX.RegisterServerCallback('esx_mechanicjob:getPlayerInventory', function(source
 
 	cb({items = items})
 end)
+
```

Comment:
These callbacks are never been called in any client scripts, therefore, its functions will not be executed.

Reply:
These callbacks were removed po sa commit #3, this is outdated po